typescript/sdk/src/tokenBridge.ts

```diff
@@ -0,0 +1,26 @@
+import {
```

Comment:
Consider moving this to `/middleware` ? Or making a tokenBridge dir and keeping everything together?